packages/vue-app/template/server.js

```diff
@@ -51,8 +51,8 @@ Vue.component(NuxtLink.name, NuxtLink)
 
 const noopApp = () => new Vue({ render: h => h('div', { domProps: { id: '<%= globals.id %>' } }) })
 
-function urlJoin () {
-  return Array.prototype.slice.call(arguments).join('/').replace(/\/+/g, '/')
+function urlJoin (base, ...args) {
+  return joinURL(base, ...args)
```

Comment:
Is this an alias? :D Then maybe we can directly rexport it to prevent refactoring in other usages?

Reply:
I will prepare separate PR with more utils from `ufo`